src/tests/List/VideoList.spec.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import VideoList from '../../components/List/VideoList';
+
+describe('List component test', () => {
+  it('testing is generate href of video', () => {
```

Comment:
You can change your description and usage of `it` to `test`
```suggestion
  test('the generation of a video link', () => {
```
Thus others can read it like:
_Test the generation of a video link_